src/main/java/pl/edu/mimuw/chess/Gracz.java

```diff
@@ -0,0 +1,54 @@
+package pl.edu.mimuw.chess;
+
+import pl.edu.mimuw.Figura;
+
+import java.util.ArrayList;
+import java.util.Objects;
+import java.util.Random;
+
+public class Gracz {
+  public final Kolor kolor;
```

Comment:
tu widze wszedzie jednorodne podejscie przyjales przynajmniej, ale w wiekszosci to jest blad, bo pozwala na nieoczekiwane modyfikacje pol tych obiektow